doc/data/messages/d/deprecated-argument/details.rst

```diff
@@ -1,2 +1,18 @@
-The actual replacement need to be studied on a case by case basis
-by reading the deprecation warning or the release notes.
+In order to be raised, this message need a custom checker like this one::
+
+    class CustomDeprecationChecker(DeprecatedMixin, BaseChecker):
+        name = "deprecated"
+
+        def deprecated_arguments(
+            self, method: str
+        ) -> tuple[tuple[int | None, str], ...] | tuple[tuple[int, str], tuple[int, str]]:
```

Comment:
I think an example is not clear out of blue. I need to go back to source code documentation in order to understand it. We should put here a link to documentation of parameters and return value (or at least put a link to an example https://github.com/pylint-dev/pylint/blob/main/examples/deprecation_checker.py)  or document it right here.

Reply:
I forgot about this example, how about only a link to the example ? I also was not very clear how it should be done. Do we agree that the only way is to create a plugin, there's no configuration that can make deprecate-arguments be raised otherwise, right ?

Reply:
> there's no configuration that can make deprecate-arguments be raised otherwise, right ?

I think I'm missing some context here, what is the reason we can't just have a vanilla example that uses one of the deprecated arguments in `stdlib.py`?

Reply:
No I thougth the same. But after a 10mn search I found no such example of deprecated arguments in the stdlib (?!). Also it seems that we only have a base class that permit to create a custom checker  and not an actual builtin pylint checker ? (I did not find anything in pylint either). This is what I'm trying to confirm with @matusvalo in fact :)

Reply:
👀 https://github.com/pylint-dev/pylint/blob/9c1c6637b3ea19bcd53a4efff7df9e178323f51c/pylint/checkers/stdlib.py#L85-L88

Reply:
Weeeell, I just feel silly now.

Reply:
Now I am not sure whether everything was answered or there is still something waiting for me... :-D

Reply:
Any idea why we can't add values in ``DEPRECATED_ARGUMENTS`` via options/configuration instead of creating a custom checker @matusvalo ? Thank you for initially reviewing the PR :)

Reply:
> Any idea why we can't add values in DEPRECATED_ARGUMENTS via options/configuration instead of creating a custom checker @matusvalo ?

It depends on the scope I suppose. If the scope is *just* python I think sure it can be via configuration I suppose. But if you want to have it as generic checker to be used for other projects - e.g. for django the plugin developer needs to specify logic how you get framework version in order to compare it to the configuration.